examples/digital_fingerprinting/production/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 ARG MORPHEUS_CONTAINER=nvcr.io/nvidia/morpheus/morpheus
-ARG MORPHEUS_CONTAINER_VERSION=v22.09.00-runtime
+ARG MORPHEUS_CONTAINER_VERSION=v22.11.00-runtime
```

Comment:
Shouldn't this be 23.01?

Reply:
welllll
It's code and with NGC publishing lagging behind the GitHub release, in theory someone could stub their toe trying to build the starter image from the missing image reference.

Reply:
@mdemoret-nv thoughts?

Reply:
I think it should be 23.01. Correct me if I am wrong @dagardner-nv but isnt there a section in the DFP documentation which instructs the user to build the container?

Reply:
I'm going to go with 23.01. Found in the documentation this:
```
## Build the Morpheus container
This is necessary to get the latest changes needed for DFP. From the root of the Morpheus repo:
```bash
./docker/build_container_release.sh
```
```
Because of that, you want the version to match the release